website/docs/customization/plugin/plugin_only.md

```diff
@@ -0,0 +1,70 @@
+---
+id: plugin_only
+description: The Plugin Only Mode
+slug: /plugin/plugin_only
+---
+
+# The Plugin-Only Mode
+
+The plugin-only mode is a restricted mode of TaskWeaver that only allows you to use plugins. 
+Compared to the full mode, the plugin-only mode has the following restrictions:
+
+1. The generated code only contains the calls to the plugins. 
+   For example, the following code only calls the `ascii_render` plugin and does not contain any "free-form" code.
+    ```python
+    r1=ascii_render(text="Hello")
+    r1
+    ```
+    
+2. Only the plugins with `plugin_only: true` in the yaml file will be loaded. 
+   For example, the following plugin will be loaded in the plugin-only mode:
+    ```yaml
+    name: ascii_render
+    code: ascii_render
+    plugin_only: true
+    ...
+    ```
+   If this field is not specified, the default value is `false`.
+   For these plugins, the argument type can only be `str`, `int`, `boolean`, or `float`.
```

Comment:
"For these plugins" -> For plugins in the plugin-only mode, we only support arguments of type `str`...

Reply:
done